### PR TITLE
remove baseprice column

### PR DESCRIPTION
**Pull request by jochenmanz**
## Description
Please describe your pull request:
* Why is it necessary?
The baseprice column was removed from the price model in 5.2. This is only a small cleanup PR.
* What does it improve?
Database size
* Does it have side effects?
none

| Questions        | Answers
| ---------------- | -------------------------------------------------------
| BC breaks?       | no
| Tests pass?      | yes
| Related tickets? | none
| How to test?     | reinstall shopware from scratch or update an existing 5.2 installation. No baseprice column should be present in s_articles_prices

### CODE REVIEW

File: _sql/install/latest.sql [hunk 1]
```diff
@@ -360,7 +360,6 @@ CREATE TABLE IF NOT EXISTS `s_articles_prices` (
   `articledetailsID` int(11) NOT NULL DEFAULT '0',
   `price` double NOT NULL DEFAULT '0',
   `pseudoprice` double DEFAULT NULL,
-  `baseprice` double DEFAULT NULL,
```
Comment (mitelg) [inline _sql/install/latest.sql]: don't change the install/latest.sql

**Review by mitelg — CHANGES_REQUESTED** [review]

**Comment by mitelg** [comment]
the builds are failing, because you changed the latest.sql :wink:

**Review by mitelg — APPROVED** [review]

File: _sql/migrations/815-remove-base-price-column.php [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+class Migrations_Migration815 extends Shopware\Components\Migrations\AbstractMigration
+{
+    public function up($modus)
+    {
+        $statement = $this->connection->query('SELECT * FROM s_articles_prices');
```
Comment (shyim) [inline _sql/migrations/815-remove-base-price-column.php]: show columns FROM s_articles_prices would be better here :)

**Review by shyim — CHANGES_REQUESTED** [review]

**Comment by dnoegel** [comment]
If we merge this, I'd suggest 5.3 due to the schema change.

**Comment by samuelvogel** [comment]
Just for reference, why the field is currently still there: https://github.com/shopware/shopware/pull/527#issuecomment-218442872

**Comment by mitelg** [comment]
@dnoegel yeah, we already discussed that. and we will do it on 5.3. IF we do it :wink: :smile:

**Pull request closed without merging**